src/server/scripts/Zandalar/Underrot/boss_cragmaw_the_infested.cpp

```diff
@@ -0,0 +1,377 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AreaTrigger.h"
+#include "AreaTriggerAI.h"
+#include "CreatureAI.h"
+#include "CreatureAIImpl.h"
+#include "InstanceScript.h"
+#include "ObjectAccessor.h"
+#include "ScriptMgr.h"
+#include "ScriptedCreature.h"
+#include "SpellScript.h"
+#include "SpellAuras.h"
+#include "SharedDefines.h"
+#include "TemporarySummon.h"
+#include "underrot.h"
+
+enum CragmawSpells
+{
+    // Cragmaw
+    SPELL_CRAWG_EATING                  = 279156,
+    SPELL_TANTRUM_ENERGY_BAR            = 271771,
+    SPELL_TANTRUM_PERIODIC_ENERGY       = 271775,
```

Comment:
Rename to SPELL_POWER_ENERGIZE_TANTRUM. The whole point of renaming is to allow the reader to know SPELL_POWER_DISPLAY_TANTRUM is the SPELL_AURA_MOD_OVERRIDE_POWER_DISPLAY and to avoid confusions in SpellScript names since you called this one like SPELL_TANTRUM_PERIODIC_ENERGY whereas the SpellScript attached to it is spell_tantrum_energy_bar.